### ISSUE TITLE
[BUG] Placeholder for pasword in URL is always visible

### ISSUE THREAD

**Issue by ptar** [PhilippC/keepass2android#2452]
### Checks

- [X] I have read the FAQ section, searched the open issues, and still think this is a new bug.

### Describe the bug you encountered:

* Use the {PASSWORD} placeholder in the url
   * E.g. http://{USERNAME}:{PASSWORD}@...
* Save the change
* You can see the password as clear text in the url field

### Describe what you expected to happen:

The password should not be visible in the url (but usable)

### What version of Keepass2Android are you using?

Version 1.09e

### Which version of Android are you on?

10

**Comment by PhilippC** [comment]
is any other KeePass client handling this in a different way?

**Comment by ptar** [comment]
Yes, e.g. KeepassXC (for Windows) shows `http://{USERNAME}:{PASSWORD}@...` as `http://someuser:******@...`

I would expect something similar in Keepass2Android (and BTW in any other Keepass-client)
